universal-login-react/src/ui/commons/GasPrice/TransactionFeeChoose.tsx

```diff
@@ -43,7 +42,7 @@ export const TransactionFeeChoose = ({gasModes, gasLimit, onGasOptionChanged, mo
                 <div className="transaction-fee-details">
                   <div>
                     <p className="transaction-fee-amount-usd">{calculateTransactionFee(usdAmount, gasLimit)} USD</p>
-                    <p className="transaction-fee-amount">{formatCryptoBalance(safeMultiply(option.gasPrice, gasLimit), 9)} {option.token.symbol}</p>
+                    <p className="transaction-fee-amount">{ValueRounder.floor(safeMultiply(option.gasPrice, gasLimit))!} {option.token.symbol}</p>
```

Comment:
Why `!`?

Reply:
This function did not work correctly before, we've decided with Jarek, that we can remove it and use my option